ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py

```diff
@@ -72,7 +72,8 @@
     required: False
     type: str
     description:
-    - Name of the snapshot this access control record applies to.
+    - Name of the snapshot this access control record applies to. If this record applies to a VVol snapshot, this attribute is required.
+      Otherwise, this attribute is not meaningful and should not be specified.
```

Comment:
vVols should only be managed by the VASA provider on array, never externally.  Let's drop the reference to protocol_endpoint above and the additional text you added here.